paint.js

```diff
@@ -585,10 +586,110 @@ PaintCanvasMorph.prototype.init = function (shift) {
         var key = this.world().currentKey;
         return (key === 16);
     };
+    // should we calculate the center of the image ourselves,
+    // or use the user position
+    this.automaticCrosshairs = true;
     this.buildContents();
 };
 
+// Returns a rectangle that encloses all the non-transparent pixels on the canvas.
```

Comment:
nice! Can we maybe reuse the existing method from Costume (here: https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/blob/master/objects.js#L6333) that (I believe) already solves the bounding box?